rpcserver.go

```diff
@@ -648,8 +648,14 @@ func handleCreateRawTransaction(s *rpcServer, cmd interface{}, closeChan <-chan
 				"or stake")
 		}
 
+		utxo, err := s.chain.FetchUtxoEntry(txHash)
```

Comment:
This will break the intended behavior of `createrawtransaction` which is to also allow callers to create transactions for which utxos are not yet known.  This information needs to be provided by the caller.